apps/researcher/src/lib/uri-transformer.test.ts

```diff
@@ -0,0 +1,26 @@
+import {describe, expect, it} from '@jest/globals';
+import {encodeUri, decodeUri} from './uri-transformer';
+
+describe('encodeUri', () => {
+  it('encodes an url', () => {
+    const encodedUri = encodeUri('https://museum.example.org/');
+    expect(encodedUri).toBe('https%3A%2F%2Fmuseum%252Eexample%252Eorg%2F');
+  });
+
+  it('encodes a string with spaces', () => {
```

Comment:
Spaces have their own tests. Perhaps add a comment why?

Reply:
No real reason. I just found it hard to think of things to test for one line of code.